_commands/basics/tail.md

```diff
@@ -3,22 +3,42 @@
 
 tail
 -------
-TODO: Add documentation for this command by submitting a pull request.
+`tail` is used to read the final few lines of text from a document. The default length is 10. 
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ tail example.txt
+$ tail example1.txt example2.txt
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `tail -n`
+
+~~~ bash
+$ tail -n 15 example.txt
+~~~
 
 ##### Break it down
+ * The `n` flag specifies the number of lines that tail will print.
+
+#### `tail -c`
+
+~~~bash
+$ tail -c 500 example.txt
+~~~
 
-#### Example command
+##### Break it down
+ * The `c` flag allows you to only print the last k amount of bytes.
+
+#### `tail -f`
+
+~~~ bash
+$ tail -f example.txt
+~~~
 
 ##### Break it down
+ * The `f` flag follows the file and outputs any new text added to the end of the file.
```

Comment:
I'm not sure what you mean here, add sample output to show what it actually does.